M/MAGEMin/build_tarballs.jl

```diff
@@ -3,47 +3,34 @@
 using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
 const YGGDRASIL_DIR = "../.."
-include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "MAGEMin"
-version = v"1.7.2"
-
-MPItrampoline_compat_version="5.2.1"  
+version = v"1.7.3"
 
 # Collection of sources required to complete build
 sources = [GitSource("https://github.com/ComputationalThermodynamics/MAGEMin", 
-                    "d4f464bc1cf7cf165faa980ff3694ae596f45f41")                 ]
+                    "98ede02c4634a3bd0a1860d33f46a36be678ee98")                 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd MAGEMin*
 
-if [[ "${target}" == *-mingw* ]]; then
-    MPI_LIBS="-lmsmpi"
-elif grep -q MPICH_NAME $prefix/include/mpi.h; then
-    MPI_LIBS="-lmpi"
-elif grep -q MPItrampoline $prefix/include/mpi.h; then
-    MPI_LIBS="-lmpitrampoline"
-elif grep -q OMPI_MAJOR_VERSION $prefix/include/mpi.h; then
-    MPI_LIBS="-lmpi"
-fi
-
 CCFLAGS="-O3 -g -fPIC -std=c99"
 
 if [[ "${target}" == *-apple* ]]; then 
     # Use Accelerate for Lapack dependencies
-    LIBS="-L${libdir} -lm -framework Accelerate -lnlopt ${MPI_LIBS}"
+    LIBS="-L${libdir} -lm -framework Accelerate -lnlopt"
     INC="-I${includedir}"
 else
-    LIBS="-L${libdir} -lm -lopenblas -lnlopt ${MPI_LIBS}"
+    LIBS="-L${libdir} -lm -lopenblas -lnlopt"
     INC="-I${includedir}"
 fi
 
 # Compile library:
-make -j${nproc} CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" lib
+make CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" lib
 
 # Compile binary
-make -j${nproc} EXE_NAME="MAGEMin${exeext}" CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" all
+make EXE_NAME="MAGEMin${exeext}" CC="${CC}" CCFLAGS="${CCFLAGS}" LIBS="${LIBS}" INC="${INC}" all
```

Comment:
Why removing `-j${nproc}`?  That'd tend to make compilation a lot slower.

Reply:
I see, I thought this had to do something with MPI here